generator/generator.yml

```diff
@@ -400,3 +400,43 @@ modules:
         type: EnumAsStateSet
     auth:
       community: raritan_public
+
+# InfraPower PDU's
+#
+# https://www.austin-hughes.com/support/software/infrapower/IPD-MIB.7z
+  infrapower_pdu:
+    walk:
+      - sysUpTime
+      - 1.3.6.1.4.1.34550.20.2.1.1.1.1 # pduIndex
+      - 1.3.6.1.4.1.34550.20.2.1.1.1.6 # pduName
+      - 1.3.6.1.4.1.34550.20.2.1.1.1.7 # pduLocation
+      - 1.3.6.1.4.1.34550.20.2.1.1.1.13 # pduMainLoadVoltage
+      - 1.3.6.1.4.1.34550.20.2.1.1.1.14 # pduMainLoadAmp
+      - 1.3.6.1.4.1.34550.20.2.1.1.1.17 # pduMainActivePower
+    lookups:
+      - source_indexes: [pduIndex]
+        lookup: pduName
+      - source_indexes: [pduIndex]
+        lookup: pduLocation
+    overrides:
+      pduIndex:
+        ignore: true # Lookup metric
+      pduName:
+        ignore: true # Lookup metric
+      pduLocation:
+        ignore: true # Lookup metric
+      pduMainLoadAmp:
+        regex_extracts:
+          Fix:
+            - regex: '(.*)'
+              value: '$1'
+      pduMainLoadVoltage:
```

Comment:
IIrc some of these were multiplied by 10, we could fix the units here to actually be volts/amps with a little bit of regex.

Reply:
My regex is pretty terrible, i just did the adjustment in grafana for myself, wouldn't mind it being changed in here however.

Reply:
Something like `(.*)(.)` with a replacement of `$1.$2`

Reply:
Got the 10's done, but the 10000 may be tricky as it may not have the amount of digits
eg
```
0000000 = 00.00000kW
0000025 = 00.00025kW
0000123 = 00.00123kW
0001234 = 00.01234kW
0028819 = 00.28819kW
0123456 = 01.23456Kw
1234567 = 12.34567kW```

Reply:
Does it always have at least 5 digits?

Reply:
> Does it always have at least 5 digits?

In our current environment it does, but that may not hold true for all setups.

Reply:
Sorry its a /10000 not a /1000

Reply:
We can do it the simple way then in one extract.

Reply:
Been a long day..  Appoligies / 100000